### ISSUE TITLE
Service "auth" not found

### ISSUE THREAD

**Issue by markszenteTB** [micro/micro#1691]
**Describe the bug**
After running `micro server` and entering any subsequent command (for example, `micro services`), the `Service "auth" not found` message is printed. 

`micro login` returns the following message. The same JSON is returned by the HTTP API interface:

```
Error authorizing request: {"Id":"go.micro.client","Code":500,"Detail":
"{\"Id\":\"go.micro.client\",\"Code\":500,\"Detail\":\"service auth: route not found\",\"Status\":\"Internal Server Error\"}","Status":"Internal Server Error"}
``` 

**To Reproduce**
1. I have been following the [tutorial](https://micro.mu/getting-started). 
2. Start Micro by `micro server`
3. Enter `micro services`, or try to login by `micro login`
4. See the respective errors.

**Expected behavior**
- As per docs, `micro login` should print `Successfully logged in.`, provided the credentials are correct. 
- `micro services` should list the running services.

**Screenshots**
The result of `micro server`:
![image](https://user-images.githubusercontent.com/62751480/106059050-681cc480-60f2-11eb-88d5-8b565412ea8a.png)
The results of `micro services`:
![image](https://user-images.githubusercontent.com/62751480/106061643-f6df1080-60f5-11eb-9c69-9a4e9da34f00.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: 10.0.19042 Build 19042
 - Micro version: v3.0.4

**Additional context**
I'm pretty sure it's an issue with mDNS. I have updated Windows to this version (10.0.19042), which supposedly have native mDNS support, tried setting `EnableMulticast` in `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows NT\DNSClient`, installing Bonjour, etc. However, I'm totally new to Go, microservices and Micro and not that experienced in the low level networking stuff... I would be happy to use my Mac, but Go doesn't work with the M1 processors yet, as far as I know.
So I'm stuck.

**Comment by crazybber** [comment]
https://github.com/micro/micro/discussions/1650

**Comment by baina-echo** [comment]
I aso had this problem. Can anyone help me?

**Comment by liuaiyuan** [comment]
I also encountered this problem, MacOS

**Comment by djsxianglei** [comment]
me too

**Comment by mail2thota** [comment]
Hi, I am newbie to micro and got stuck with the same issue. Any solution on this?

**Comment by dashessmith** [comment]
now ?

**Comment by mail2thota** [comment]
@dashessmith yes I cleared the micro binary and do a go get again. It sill the same

**Comment by dashessmith** [comment]
@mail2thota i not a contributer, i got the same issue

**Comment by wwhai** [comment]
I have fond the problem,It at `service.go line 136`:
```go
	p, err := s.Process.Fork(s.Exec)
	if err != nil {
		s.Status(runtime.Error, err)
		return err
	}
```

When micro stoped, Those process will out of control which started by micro service module. So I think should add a global process manager for micro service module to adapter `stop` or `restart`.
